### DISCUSSION TITLE
submit a form rendered with embedded controller

### DISCUSSION THREAD

**Discussion by butachan** [symfony/symfony#47266] [Q&A]
Hello, 
I use embedding controller to render on the same page 2 twigs : 
the first twig renders every review
the second twig renders the form to submit a review. This is done with `{{ render(path('app_new_review',{id:product.id})) }}` 
```php
    #[Route('/newreview/{id}', name: "app_new_review")]
    public function newreview(Product $product,Request $request,EntityManagerInterface $entityManager):Response
    {
     $avis = new Review();
...
        $form = $this->createForm(AvisType::class,$avis);
        $form->handleRequest($request);
        if ($form->isSubmitted() && $form-> isValid()){
            $entityManager->persist($avis);
            $entityManager->flush();
            return $this->redirectToRoute('app_avis',['id'=>$product->getId()]);
        }
        return $this->renderForm('review/_form.html.twig', [
            'reviewform' => $form,
        ]);
     }
```
is correct. I tried it on an independant page and it works.

But when I put with "all reviews" twig, it won't submit. 
Actually, I tried dd("foo");  in the `newreview` function but it seems that the function isn't executed.

I d like to keep the embedded system so that my "indexreview" function and my allreviews twig are not overloaded.

Can someone help me figuring out how to use/submit a form included in twig with render(path()) ?

**Comment by Asmitta-01** [comment]
So you are saying that the form is rendered but not submitted ?
Did you check the action attribute of the form ? If you didn't specify it in the AvisType maybe it should be empty.


_Evening_

**Reply by butachan** [reply]
yes, that's it.
I was told to avoid render(path()) so in the end i put all reviews and new review in the same controller

**Reply by Asmitta-01** [reply]
And what about the action attribute as i and @stof asked ?

**Comment by stof** [comment]
I cannot be sure without seeing your template rendering the form. But could it be that the form is rendered with an empty action attribute, which would make it submit to the current URL, which is *not* the URL of `app_new_review` route (as it was embedded into another page).
